nvflare/private/fed/app/client/worker_process.py

```diff
@@ -32,23 +34,37 @@
 from nvflare.private.fed.client.command_agent import CommandAgent
 
 
+def check_parent_alive(parent_pid, stop_event: threading.Event):
+    while True:
+        if stop_event.is_set():
+            break
+        # TODO: this check only valid in UNIX
```

Comment:
According to the documentation, this works on Windows since 3.2.

Reply:
Hi @nvidianz,
`os.getppid()` works on Windows that is correct.
But there is a note (https://docs.python.org/3/library/os.html#os.getppid): 

"When the parent process has exited, on Unix the id returned is the one of the init process (1), on Windows it is still the same id, which may be already reused by another process."

I did find a solution to this though, since we already included psutil.
I can use  `psutil.pid_exists`, let me change this.